kubeconfig

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
whoops?

Reply:
and the clusterctl binary too

Reply:
fixed sry :)

Reply:
was trying to seperate deps from code changes